### PR TITLE
US303 Ascii Download Relocation

### PR DESCRIPTION
**Pull request by Tengelma**
### Overview
This PR contains changes relevant to Ascii data exports.

The big stuff:
- Introduces a new endpoint 'get_object_data' that takes a list of parameters and returns a JSON containing the vectors associated with those parameters for all objects in the session
- Introduces downloadData function on the client, which uses several calls to above endpoint in order to build a zip archive of TSV files in the format used by the current site

The little stuff:
- Some renaming to make things a little more explicit in what they are / what they do
- Moved download functions out of Download.vue and into a new file in utils, 'downloads.js'
- Deleted all stuff related to ASCII & keymap on server
- Tests
- Reverted astropy to 4.1 because pip could not find 4.2 when I tried to install it. Functionality seems identical for our purposes so hopefully this is ok

### Includes:
See files changed. Most of the important ones have been mentioned already.

### Developer Checklist:
- [X] The code follows the Quality Policy file on Google Drive
- [X] My code has been developer-tested and includes unit tests
- [X] I have considered proper use of exceptions
- [X] I have eliminated IDE warnings
- [X] I have included tasks associated with this pull request in the Sprint Progress Sheet.

### Notes:
I opted against sending labels to server and building the archive there because
- the server doesn't need to be aware of display details
- we'd basically need to send the entirety of PLOT_AXIS_OPTIONS and PLOT_AXIS_METADATA for it to work, and at that point we may as well just leave the KEYMAP in
- this gives us A LOT more freedom to customize what data the user can request / receive.

### CODE REVIEW

File: client/yarn.lock
Comment (aschnapp) [inline client/yarn.lock:6166]: We should add this file to gitignore

File: server/requirements.txt
Comment (aschnapp) [inline server/requirements.txt:2]: why?

**Review by aschnapp — CHANGES_REQUESTED** [review]
why change the dependency?

File: client/yarn.lock
Comment (aschnapp) [reply client/yarn.lock:6166]: This is just a comment. Won't reject the pr based on this.

File: server/requirements.txt
Comment (Tengelma) [reply server/requirements.txt:2]: image

Mentioned this in the PR. Not sure what the solution is.

Comment (aschnapp) [reply server/requirements.txt:2]: Ok. We can just use the old one

**Review by aschnapp — APPROVED** [review]

**Review by aneuhold — APPROVED** [review]
Looks good

**Comment by aschnapp** [comment]
@aneuhold
Before you guys merge, are guys having issues with astropy 4.1? 
I cannot install it with pip over here.

**Comment by aschnapp** [comment]
@Tengelma don't merge yet.

**Comment by aneuhold** [comment]
> @aneuhold
> Before you guys merge, are guys having issues with astropy 4.1?
> I cannot install it with pip over here.

I did not have an issue installing astropy 4.1

**Comment by aneuhold** [comment]
This is probably good to merge unless @aschnapp is still having issues. And after it has dev merged into it first.

**Pull request merged**
